packages/lib-classifier/src/components/Classifier/Classifier.spec.js

```diff
@@ -0,0 +1,431 @@
+    before(function () {
+      const subject = SubjectFactory.build({ locations: [{ 'image/png': 'https://foo.bar/example.png' }] })
+      const store = mockStore({ subject })
+      const project = store.projects.active
+      const projectSnapshot = { ...getSnapshot(project) }
+      workflow = store.workflows.active
+      const workflowSnapshot = { ...getSnapshot(workflow) }
+      render(
+        <Classifier
+          classifierStore={store}
+          project={projectSnapshot}
+          subjectID={subject?.id}
+          workflowID={workflowSnapshot?.id}
+          workflowSnapshot={workflowSnapshot}
+          workflowVersion={workflowSnapshot?.version}
+        />,
+        {
+          wrapper: withStore(store)
+        }
+      )
+      taskTab = screen.getByRole('tab', { name: 'TaskArea.task'})
+      tutorialTab = screen.getByRole('tab', { name: 'TaskArea.tutorial'})
+      subjectImage = screen.getByRole('img', { name: `Subject ${subject.id}` })
+      tabPanel = screen.getByRole('tabpanel', { name: '1 Tab Contents'})
+      const task = workflowSnapshot.tasks.T0
+      const getAnswerInput = answer => within(tabPanel).getByRole('radio', { name: answer.label })
+      taskAnswers = task.answers.map(getAnswerInput)
+    })
+
+    it('should have a task tab', function () {
+      expect(taskTab).to.be.ok()
+    })
+
+    it('should have a tutorial tab', function () {
+      expect(tutorialTab).to.be.ok()
+    })
+
+    it('should have a subject image', function () {
+      expect(subjectImage).to.be.ok()
+    })
+
+    describe('task answers', function () {
+      it('should be displayed', function () {
+        expect(taskAnswers).to.have.lengthOf(workflow.tasks.T0.answers.length)
+      })
+
+      it('should be linked to the task', function () {
+        taskAnswers.forEach(radioButton => {
+          expect(radioButton.name).to.equal('T0')
+        })
+      })
+
+      it('should be disabled', function () {
+        taskAnswers.forEach(radioButton => {
+          expect(radioButton.disabled).to.be.true()
+        })
+      })
+    })
+  })
+
+  describe('after the subject has loaded', function () {
+    before(function () {
+      const subject = SubjectFactory.build({ locations: [{ 'image/png': 'https://foo.bar/example.png' }] })
+      const store = mockStore({ subject })
+      const project = store.projects.active
+      const projectSnapshot = { ...getSnapshot(project) }
+      workflow = store.workflows.active
+      const workflowSnapshot = { ...getSnapshot(workflow) }
+      render(
+        <Classifier
+          classifierStore={store}
+          project={projectSnapshot}
+          subjectID={subject?.id}
+          workflowID={workflowSnapshot?.id}
+          workflowSnapshot={workflowSnapshot}
+          workflowVersion={workflowSnapshot?.version}
+        />,
+        {
+          wrapper: withStore(store)
+        }
+      )
+      store.subjectViewer.onSubjectReady()
```

Comment:
This line is the difference from the previous test. It mocks the image load event. Maybe there’s a better way to do that? I’m not sure.

Reply:
I’m pretty sure this can be merged into the previous test suite, so we update the store then test the output, rather than running the same setup all over again.

Reply:
Really good to know. Non-blocking suggestion that we add a note in this test suite about why `onSubjectReady()` and `rerender` can't be used in this scenario.

Reply:
This should use something like `fireEvent.load(subjectImage)`, rather than directly manipulating the store. We want to check that you can classify after the subject has loaded, so that we can catch bugs like the one that was fixed by #3250. Also, that bug doesn’t have an issue. That might be useful for tracking regressions.

Reply:
`rerender` is used to simulate prop changes. Props don’t change when a subject loads so `rerender` isn’t really relevant here. Mentioning it might be confusing. 

I need to rewrite these tests to fire events after the classifier mounts.